es6-classes/circle.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(area, circumference, radius) {
+    super(area, circumference, radius);
+    this.radius = radius;
+  }
+
+  getValues() {
+    console.log(`My radius is ${this.radius}`);
```

Comment:
You need to call `super.getValues()` to get the parent values.

This should return a string, not log it.